### ISSUE TITLE
eget-1.0.0-linux_amd64 incompatible with Citrix Hypervisor

### ISSUE THREAD

**Issue by thorstenkampe** [zyedidia/eget#19]
```
eget --version
eget: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by eget)

lsb_release -sd
"Citrix Hypervisor release 8.2.0 (xenenterprise)"
```

Previous version 0.3.1 worked fine. I guess a `musl` (or static) version would work.

**Comment by zyedidia** [comment]
Thanks I have released a new version that uses static linking to avoid this problem.

**Comment by thorstenkampe** [comment]
The same issue has reoccured on Ubuntu 20.04 (fully patched) with eget 1.3.0:
```
> eget --version
eget: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by eget)

> lsb_release -sd
Ubuntu 20.04.5 LTS
```

**Comment by zyedidia** [comment]
I have uploaded new binaries for 1.3.0 and also changed the build script this time so I won't forget about it again.
